### ISSUE TITLE
Steps gets divided by 4 in img2img in web ui

### ISSUE THREAD

**Issue by 19wolf** [invoke-ai/InvokeAI#858]
**Describe your environment**
- GPU: MPS
- VRAM: [if known]
- CPU arch: arm
- OS: macOS
- Python: Miniconda? I think? 3.10.5
- Branch: Development
- Commit: 779422d01b848e5643a5d8727e5568002b936937

**Describe the bug**
When running Img2Img, the number of steps specified in the GUI gets divided by 4.

**To Reproduce**
1. Use img2img in the gui2. 
2. Number of steps specified in the sampler section gets divided by 4 when run. 

**Expected behavior**
Number of steps specified should be the same that runs

**Additional context**
My run command: `PYTORCH_ENABLE_MPS_FALLBACK=1 python scripts/dream.py --web`
I have k_lms selected as the sampler
From the command line:
```
>> Image generation requested: {'prompt': 'fire dance', 'iterations': 3, 'steps': 64, 'cfg_scale': 8, 'height': 512, 'width': 512, 'sampler_name': 'k_lms', 'seed': 4236713110, 'seamless': False, 'progress_images': False, 'init_img': 'outputs/init-images/input.jpg', 'strength': 0.3, 'fit': True, 'variation_amount': 0}
ESRGAN parameters: False
GFPGAN parameters: False
[[directory]]/InvokeAI/ldm/modules/embedding_manager.py:155: UserWarning: The operator 'aten::nonzero' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at  /Users/runner/work/_temp/anaconda/conda-bld/pytorch_1659484612588/work/aten/src/ATen/mps/MPSFallback.mm:11.)
  placeholder_idx = torch.where(
>> loaded input image of size 499x750 from [[directory]]/InvokeAI/outputs/img-samples/init-images/input.jpg
>> image will be resized to fit inside a box 512x512 in size.
>> after adjusting image dimensions to be multiples of 64, init image is 320x512
>> sampler 'KSampler' is not yet supported. Using DDIM sampler
Generating:   0%|                                                                                                                                                                                                                                                       | 0/3 [00:00<?, ?it/s]Running DDIM Sampling with 19 timesteps
Decoding image: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 19/19 [00:12<00:00,  1.52it/s]
```

**Comment by PutUsernameHere** [comment]
I'm pretty sure that when you use Img2Img, it multiplies the strength (in your case 0.3) by your target samples for the real sample amount. This makes sense when you're iterating the strength since the lower the strength, the less you likely need to spend time on the new image.

**Comment by psychedelicious** [comment]
Yep, this is how it works internally.

Perhaps it's better if we represent the UI steps chooser as the "real" steps the generation should do? This would be, conceptually, a breaking change, and past generations' steps variable would be incorrect. I think it's better to keep it how it is.

**Comment by bbeus88** [comment]
I think a simple solution would be to change the label.  Instead of "Strength" it should say "Variation Amount" similar to the label under "Seed and Variation".

I think this makes sense since a higher number means a bigger change.

Also, adding some additional text in the help tooltip about how this number affects the number of steps might not be a bad idea.
